internal/host/transition_test.go

```diff
@@ -213,31 +215,59 @@ var _ = Describe("RegisterHost", func() {
 
 	Context("register after reboot", func() {
 		tests := []struct {
-			name     string
-			srcState string
-			progress models.HostProgressInfo
+			name          string
```

Comment:
I saw that you are not using the name anymore and instead you use: mt.Sprintf ... so why do you need this line?